mmaction/models/backbones/st_gcn.py

```diff
@@ -0,0 +1,449 @@
+import numpy as np
```

Comment:
This file needs some reconstruction: we can have a directory named `SkeletonGCN`, a base GCN class, while STGCN is the children class of the base class.